x/transfermiddleware/keeper/keeper.go

```diff
@@ -167,7 +167,16 @@ func (k Keeper) HasAllowRlyAddress(ctx sdk.Context, rlyAddress string) bool {
 	store := ctx.KVStore(k.storeKey)
 	key := types.GetKeyByRlyAddress(rlyAddress)
 
-	return store.Has(key)
+	if store.Has(key) {
+		return true
+	}
+
+	prefixStore := prefix.NewStore(store, types.KeyRlyAddress)
+	iter := prefixStore.Iterator(nil, nil)
+	defer iter.Close()
```

Comment:
Why do you need this check?

Reply:
it allow permissionless relaying if no relayers registered. see original PR merged into other grant. clear for devnet needed